### PR TITLE
Add Datadog Site Regions to Code Snippets in Custom Tags and Measures

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

We tried using the site region workaround in https://github.com/DataDog/documentation/pull/22750 but that was not a good experience for two reasons: 

• there is no reason to have people to specify DD_SITE=[datadoghq.com](http://datadoghq.com/) as this is the default. If I want DD_SITE to not be in the code block for us1, I have to duplicate and have an if which is not great.
• Many people will copy paste and expect things to work. In our case, they might even commit and push code before realising it does not work. This is a very frustrating experience.

This PR adds site region partials with manual code snippets updated with the site region.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 1]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
 {{< /site-region >}}
+{{< site-region region="us3" >}}
+
+To do this, run the `tag` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=us3.datadoghq.com datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:138]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 2]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
 The following example adds the tag `go.version` to the span for the current job:
 
 {{< code-block lang="shell" >}}
-datadog-ci tag --level job --tags "go.version:`go version`"
+DATADOG_SITE=us3.datadoghq.com datadog-ci tag --level job --tags "go.version:`go version`"
 {{< /code-block >}}
 
-To create facets from the tags, click the gear icon next to the tag name in the [pipeline executions page][4],
-and then click the **create facet** option.
+To create a facet from a tag, click the Gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:152]:
```suggestion
To create a facet from a tag, click the gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 3]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
 
 {{< code-block lang="shell" >}}
-datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
+DATADOG_SITE=us3.datadoghq.com datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
 {{< /code-block >}}
 
-You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`.
-{{< site-region region="us5,us3,eu,ap1" >}}
-You must specify the [Datadog site][1] using the environment variable `DATADOG_SITE`.
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:164]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 4]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
 {{< /site-region >}}
+{{< site-region region="us5" >}}
+
+To do this, run the `tag` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.com datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:191]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 5]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+DATADOG_SITE=us5.datadoghq.com datadog-ci tag --level pipeline --tags team:backend
+{{< /code-block >}}
+
+The following example adds the tag `go.version` to the span for the current job:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=us5.datadoghq.com datadog-ci tag --level job --tags "go.version:`go version`"
+{{< /code-block >}}
+
+To create a facet from a tag, click the Gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:205]:
```suggestion
To create a facet from a tag, click the gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 6]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+
+## Add measures to pipeline traces
+
+To add numerical tags to the pipeline span or the job span, run the `measure` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=us5.datadoghq.com datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:217]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 7]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+{{< /site-region >}}
+{{< site-region region="eu" >}}
+
+To do this, run the `tag` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.eu datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:244]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 8]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+DATADOG_SITE=datadoghq.eu datadog-ci tag --level pipeline --tags team:backend
+{{< /code-block >}}
+
+The following example adds the tag `go.version` to the span for the current job:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.eu datadog-ci tag --level job --tags "go.version:`go version`"
+{{< /code-block >}}
+
+To create a facet from a tag, click the Gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:258]:
```suggestion
To create a facet from a tag, click the gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 9]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+
+## Add measures to pipeline traces
+
+To add numerical tags to the pipeline span or the job span, run the `measure` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.eu datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:270]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 10]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+{{< /site-region >}}
+{{< site-region region="ap1" >}}
+
+To do this, run the `tag` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=ap1.datadoghq.com datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:297]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 11]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+DATADOG_SITE=ap1.datadoghq.com datadog-ci tag --level pipeline --tags team:backend
+{{< /code-block >}}
+
+The following example adds the tag `go.version` to the span for the current job:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=ap1.datadoghq.com datadog-ci tag --level job --tags "go.version:`go version`"
+{{< /code-block >}}
+
+To create a facet from a tag, click the Gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:311]:
```suggestion
To create a facet from a tag, click the gear icon next to a tag name on the [Pipeline Executions page][4] and click **Create Facet**.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 12]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+
+## Add measures to pipeline traces
+
+To add numerical tags to the pipeline span or the job span, run the `measure` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=ap1.datadoghq.com datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`, and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:323]:
```suggestion
You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 13]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+{{< code-block lang="shell" >}}
+DATADOG_SITE=ap1.datadoghq.com datadog-ci measure --level job --measures "binary.size:`ls -l dst/binary | awk '{print \$5}' | tr -d '\n'`"
+{{< /code-block >}}
+
+[3]: https://app.datadoghq.com/organization-settings/api-keys
+[4]: https://app.datadoghq.com/ci/pipeline-executions
+[12]: /getting_started/site/
+{{< /site-region >}}
+
+To create a measure, click the Gear icon next to a measures name on the [Pipeline Executions page][4] and click **Create Measure**.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:342]:
```suggestion
To create a measure, click the gear icon next to a measures name on the [Pipeline Executions page][4] and click **Create Measure**.
```

**Review by urseberry — APPROVED** [review]

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 14]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+[12]: /getting_started/site/
 
-[1]: /getting_started/site/
 {{< /site-region >}}
+{{< site-region region="us5" >}}
+
+To do this, run the `tag` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.com datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
```
Comment (alxch-) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:188]:
```suggestion
DATADOG_SITE=us5.datadoghq.com datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 15]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
+
+The following example adds the tag `team` to the pipeline span.
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.com datadog-ci tag --level pipeline --tags team:backend
```
Comment (alxch-) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:90]:
```suggestion
datadog-ci tag --level pipeline --tags team:backend
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 16]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+{{< /code-block >}}
+
+The following example adds the tag `go.version` to the span for the current job:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.com datadog-ci tag --level job --tags "go.version:`go version`"
```
Comment (alxch-) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:96]:
```suggestion
datadog-ci tag --level job --tags "go.version:`go version`"
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 17]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+To create a facet from a tag, click the gear icon next to a tag name on the [Pipeline Executions page][4], and click **Create Facet**.
+
+{{< img src="ci/custom-tags-create-facet.mp4" alt="Facet creation for custom tag" style="width:100%;" video="true">}}
+
+## Add measures to pipeline traces
+
+To add numerical tags to the pipeline span or the job span, run the `measure` command:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.com datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
```
Comment (alxch-) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:108]:
```suggestion
datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 18]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
+{{< /code-block >}}
+
+You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
+
+The following example adds the measure `error_rate` to the pipeline span:
 
 {{< code-block lang="shell" >}}
-datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
+DATADOG_SITE=datadoghq.com datadog-ci measure --level pipeline --measures "error_rate:0.56"
```
Comment (alxch-) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:116]:
```suggestion
datadog-ci measure --level pipeline --measures "error_rate:0.56"
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 19]
```diff
@@ -74,66 +72,274 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
 {{< /code-block >}}
 
-You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`.
+The following example adds a measure `binary.size` to the span for the currently running job:
+
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.com datadog-ci measure --level job --measures "binary.size:`ls -l dst/binary | awk '{print \$5}' | tr -d '\n'`"
```
Comment (alxch-) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:122]:
```suggestion
datadog-ci measure --level job --measures "binary.size:`ls -l dst/binary | awk '{print \$5}' | tr -d '\n'`"
```

**Review by alxch- — COMMENTED** [review]
- there was one issue with a us5 code block
- IMO we should remove the `DATADOG_SITE` when it's `datadoghq.com` because it's the default, usually we don't add it anywhere. What do you think?

Thank you for following up on that!

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 20]
```diff
@@ -74,21 +72,19 @@ Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/do
...
-{{< /code-block >}}
+{{< site-region region="us" >}}
 
-You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY`.
+To do this, run the `tag` command:
 
-{{< site-region region="us5,us3,eu,ap1" >}}
-You must specify the [Datadog site][1] using the environment variable `DATADOG_SITE`.
+{{< code-block lang="shell" >}}
+DATADOG_SITE=datadoghq.com datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
```
Comment (alai97) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:82]:
```suggestion
datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
```

**Review by urseberry — APPROVED** [review]
This version is much cleaner!

**Pull request merged**